homeassistant/components/homekit/accessories.py

```diff
@@ -16,6 +20,28 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+def add_timeout(func):
```

Comment:
At the moment, `HAP-python` creates a new thread for each added accessory. This will change in future: https://github.com/ikalchev/HAP-python/issues/64. When it's converted to async, I will change this as well.